src/server/scripts/Kalimdor/CavernsOfTime/TheBlackMorass/boss_aeonus.cpp

```diff
@@ -95,7 +94,7 @@ struct boss_aeonus : public BossAI
             DoCastVictim(SPELL_SAND_BREATH);
             context.Repeat(10s, 30s);
         })
-        .Schedule(15s, 30s, [this](TaskContext context)
+        .Schedule(1ms, [this](TaskContext context)
```

Comment:
1ms? is this correct?

Reply:
The information I was working off said "Immediately". The shortest time I could see previously used somewhere else in the core was 1ms so I used that - do we have a way to schedule something immediately without specifying a time?

Reply:
There is however I’ve never seen it being used. TaskScheduler::Async